assets/vsphere_problem_detector/12_prometheusrules.yaml

```diff
@@ -70,12 +70,25 @@ spec:
         # Using min_over_time to make sure the metric is `1` for whole 5 minutes.
         # A missed scraping (e.g. due to a pod restart) will result in prometheus re-evaluating the the alerting rule.
         expr: |
-          min_over_time(vsphere_esxi_version_total{api_version!~"6.7.3|7.*"}[5m]) > 0
+          min_over_time(vsphere_esxi_version_total{api_version=~"(^7\\.0\\.[0-1]|^6)"}[5m]) > 0
```

Comment:
Did you mean this to be - "(^7\\.0\\.[0-1])|^6" ? i.e have "^6" outside the parenthesis

Reply:
Now that you pointed it out, I think the parenthesis are not needed at all.

Reply:
Updated a bit, PTAL.